validation-test/stdlib/UnsafeBufferPointerSlices.swift

```diff
@@ -0,0 +1,635 @@
+// RUN: %target-run-simple-swiftgyb
+// REQUIRES: executable_test
```

Comment:
Remove `gyb`, add [`END.`](https://github.com/llvm/llvm-project/blob/main/llvm/docs/TestingGuide.rst#other-features)

```suggestion
// RUN: %target-run-simple-swift
// REQUIRES: executable_test
// END.
```

Reply:
I'm updating a lot of the `initialize` and `update` functions involving `some Collection` or `UnsafeMutableBufferPointer` to have the same messaging regarding overlaps. As in the existing ones, source and destination "may" or "must not" overlap. This particular function (`update(fromContentsOf: some Collection<Element>)`) happens to be the hardest to communicate about. In this case overlaps are allowable, but some pathological cases can have a correctness trap without involving undefined behaviour.